docs/content/en/4.providers/glide.md

```diff
@@ -2,18 +2,18 @@
 title: Glide Provider
 description: 'Nuxt Image has first class integration with Glide'
 navigation:
-  title: glide
+  title: Glide
 ---
 
-Integration between [glide](https://glide.thephpleague.com/) and the image module.
+Integration between [Glide](https://glide.thephpleague.com/) and the image module.
 
 To use this provider you just need to specify the base url of your service in glide.
 
 ```js{}[nuxt.config.js]
 export default {
   image: {
     glide: {
-      baseURL: 'https://glide.example.com'
+      baseURL: 'https://glide.herokuapp.com/1.0/'
```

Comment:
Note for later improvement: We probably need better documentation that this `baseURL` is a laravel application of users not a fixed base to example.

Reply:
```suggestion
      // baseURL of your laravel application
      baseURL: 'https://glide.herokuapp.com/1.0/'
```